### PR TITLE
Webhook cron

### PR DESCRIPTION
**Pull request by abdulwahidsharief**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: catalog/controller/extension/payment/razorpay.php [hunk 1]
```diff
@@ -124,14 +123,15 @@ public function index()
 
                 if ((isset($this->session->data["razorpay_order_id_" . $this->session->data['order_id']]) === false) or
                     ((isset($this->session->data["razorpay_order_id_" . $this->session->data['order_id']]) === true) and
-                    (($this->session->data["razorpay_order_amount"] === 0) or
-                    ($this->session->data["razorpay_order_amount"] !== $order_data["amount"]))))
+                        (($this->session->data["razorpay_order_amount"] === 0) or
```
Comment (yashgit891) [inline catalog/controller/extension/payment/razorpay.php:126]: Please check alignment.

File: catalog/controller/extension/payment/razorpay.php [hunk 2]
```diff
@@ -124,14 +123,15 @@ public function index()
...
+                            ($this->session->data["razorpay_order_amount"] !== $order_data["amount"]))))
                 {
                     $razorpay_order = $this->api->order->create($order_data);
 
                     $this->session->data["razorpay_order_amount"] = $order_data["amount"];
                     $this->session->data["razorpay_order_id_" . $this->session->data['order_id']] = $razorpay_order['id'];
                     $data['razorpay_order_id'] = $this->session->data["razorpay_order_id_" . $this->session->data['order_id']];
+                    $this->model_extension_payment_razorpay->addOrderForWebhook($this->session->data['order_id'], $razorpay_order['id'], 0);
```
Comment (yashgit891) [inline catalog/controller/extension/payment/razorpay.php:134]: Why are we doing this twice? Line 108 it is already added.

Comment (abdulwahidsharief) [reply catalog/controller/extension/payment/razorpay.php:133]: @yashgit891 it was for subscription removed as we are not handling subscription events

**Review by yashgit891 — APPROVED** [review]
LGTM

**Pull request merged**